README.md

```diff
@@ -1,2 +1,181 @@
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <br/>
+
+  <h3><b>Portfolio: Mobile First</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Portfolio: Mobile First <a name="about-project"></a>
+
+Portfolio: Mobile First is my first aproach at responsive web development.
+Inside this project you will find a portfolio template cloned from scratch using
+everything i've learned so far in the Microverse program.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com">HTML</a></li>
+    <li><a href="https://html.com">CSS</a></li>
+    <li><a href="https://html.com">Linters</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+The key features of this project are:
+
+- Web responsive design with a "mobile first" aproach
+- Use of flexbox to replicate a template
+- An adaptive layout that responds to screen width
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Coming soon...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need only a web browser.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/tomasesquivelgc/Portfolio_Mobile_First.git
+```
+
+### Usage
+
+To run the project, open the inde.html file.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 Tomás Esquivel
+
+- GitHub: [@githubhandle](https://github.com/tomasesquivelgc)
+- Twitter: [@twitterhandle](https://twitter.com/EsquivelTomas)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/tomas-esquivel-b2160568/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+Future features of this project include:
+
+- [ ] About, projects and contact section.
```

Comment:
About section and projects have been already introduced! 😁
You can remove them from the future features now!
You could also add what comes next in your project 😉 (desktop version)